addons/account/views/account_move_views.xml

```diff
@@ -660,7 +660,7 @@
                                 <field name="payment_reference"
                                        attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))], 'readonly': [('state', '!=', 'draft')]}"/>
                                 <field name="partner_bank_id"
-                                       context="{'default_partner_id': commercial_partner_id}"
+                                       context="{'default_partner_id': bank_partner_id}"
```

Comment:
commercial_partner_id is fine here, I think

Reply:
`bank_partner_id` is `commercial_partner_id` when `move.is_outbound()`
If we leave commercial_partner_id, bug will be the same for Vendor Refunds as with Invoices.
https://drive.google.com/file/d/1rUvpmWBnepHNrQlIQCp9YZiiH8osqxoc/view?usp=sharing
And `default_partner_id` should be the same as the domain ` domain="[('partner_id', '=', bank_partner_id)]"` otherwise newly created account will not appear in list.

Reply:
> bank_partner_id is commercial_partner_id when move.is_outbound(

ok fine then :+1: